modules/docs/arrow-docs/docs/docs/quickstart/setup/README.md

```diff
@@ -0,0 +1,216 @@
+        generatedSourceDirs += files(
+                'build/generated/source/kapt/main',
+                'build/generated/source/kapt/debug',
+                'build/generated/source/kapt/release',
+                'build/generated/source/kaptKotlin/main',
+                'build/generated/source/kaptKotlin/debug',
+                'build/generated/source/kaptKotlin/release',
+                'build/tmp/kapt/main/kotlinGenerated')
+    }
+}
+```
+
+### Maven
+ 
+#### Basic Setup
+
+Make sure to have the at least the latest version of JDK 1.8 installed.
+Add to your pom.xml file the following properties:
+```
+<properties>
+    <kotlin.version>1.3.0</kotlin.version>
+     <arrow.version>0.10.0</arrow.version>
+</properties>
+```
+
+Add the dependencies that you want to use
+```
+        <dependency>
+            <groupId>io.arrow-kt</groupId>
+            <artifactId>arrow-core</artifactId>
+            <version>${arrow.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.arrow-kt</groupId>
+            <artifactId>arrow-syntax</artifactId>
+            <version>${arrow.version}</version>
+        </dependency>
+
+```
+
+#### Enabling kapt
+
+Enable annotaton processing using kotlin plugin 
+```
+<plugin>
+    <groupId>org.jetbrains.kotlin</groupId>
+    <artifactId>kotlin-maven-plugin</artifactId>
+    <version>${kotlin.version}</version>
+    <executions>
+        <execution>
+            <id>kapt</id>
+            <goals>
+                <goal>kapt</goal>
+            </goals>
+            <configuration>
+                <sourceDirs>
+                    <sourceDir>src/main/kotlin</sourceDir>
+                </sourceDirs>
+                <annotationProcessorPaths>
+                    <annotationProcessorPath>
+                        <groupId>io.arrow-kt</groupId>
+                        <artifactId>arrow-meta</artifactId>
+                        <version>${arrow.version}</version>
+                    </annotationProcessorPath>
+                </annotationProcessorPaths>
+            </configuration>
+        </execution>
+        <execution>
+            <id>compile</id>
+            <phase>compile</phase>
+            <goals>
+                <goal>compile</goal>
+            </goals>
+            <configuration>
+                <sourceDirs>
+                    <sourceDir>src/main/kotlin</sourceDir>
+                </sourceDirs>
+            </configuration>
+        </execution>
+        <execution>
+            <id>test-compile</id>
+            <phase>test-compile</phase>
+            <goals>
+                <goal>test-compile</goal>
+            </goals>
+        </execution>
+    </executions>
+</plugin>
+```
+
+### Linting
+
+Some linters might complaint about some code practices that are common when working with functional programming. You can read more about how to solve this problem in the [Linting](http://arrow-kt.io/docs/quickstart/linting/) section.
```

Comment:
This links needs to be relative, otherwise they break. See other examples around the docs: `({{ '/docs/quickstart/libraries/' | relative_url }})`

Reply:
Oh, sorry, I copied the same content from `modules/docs/arrow-docs/docs/docs/README.md` and I didn't check it. Done!

Reply:
Yes, some of the reviews didn't flag it. There are some broken ones around to be fixed -.-

Reply:
I take the baton! I'll do an automatic replacement after finishing another task :rocket:

Reply:
There used to be a script to automatically check broken links, I cannot find it. It may have become outdated.